src/Tuura/Boolean/Parser.hs

```diff
@@ -24,17 +24,17 @@ instance Show a => Show (Expr a) where
   show (Var c)     = show c
   show (SubExpr e) = "(" ++ show e ++ ")"
 
-not :: (Expr a) -> (Expr a)
+not :: Expr a -> Expr a
 not (Not e)     = e
 not (And e1 e2) = Or (not e1) (not e2)
 not (Or e1 e2)  = And (not e1) (not e2)
 not (SubExpr e) = not e
 not (Var c)     = Not (Var c)
 
-or :: (Expr a) -> (Expr a) -> (Expr a)
+or :: Expr a -> Expr a -> Expr a
```

Comment:
Sorry, I did understand, it was just auto-capitalized :S
The `or` function is now useless.